app/test/fuzz_abci_test.go

```diff
@@ -102,6 +103,7 @@ func TestPrepareProposalConsistency(t *testing.T) {
 						true,
 						testutil.ChainID,
 						accounts[:tt.count],
+						user.SetGasLimitAndFee(1_000_000_000, 0.1),
```

Comment:
[for reviewers]
this fix could be separated out into its own PR. happy to do so if reviewers would appreciate that. originally went to go fix this issue and added the loggers simply to check if this change would fix the issue.